packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -49,7 +49,28 @@ function generateItems(items) {
   return html.join('');
 }
 
-
+function generateSelectableItems(items, context){
+
+  const html = items.map(({ content, color, selected }, index) => {
+    return `<cbp-structured-list-item ${color != 'default' ? `color="${color}"` : ''} ${selected ? `selected` : ''} >
+    <cbp-checkbox 
+      value= ${index}
+      ${context && context != 'light-inverts' ? `context=${context}` : ''}   
+    >
+      <label>
+        <input
+          type="checkbox"
+          name="checkbox"
+          value=${index}
+        />
+        <span style='display: none'>Checkbox ${index}</span>
```

Comment:
just slap a `hidden` attribute on here instead. That's what it's for.

Reply:
as discussed in IM, keeping the display: none due to bug with hidden attribute